lib/phoenix_component/declarative.ex

```diff
@@ -817,7 +817,7 @@ defmodule Phoenix.Component.Declarative do
   defp build_attrs_docs(attrs) do
     [
       "## Attributes\n",
-      for attr <- attrs, attr.doc != false, attr.type != :global, into: [] do
+      for attr <- attrs, attr.doc != false and attr.type != :global do
```

Comment:
Hm, @josevalim i didn't realise I switched from a "multiple filters" approach in the comprehension and went with `and`. I see you switched it to `&&` but should it be reverted back to multiple filters?

Reply:
It is all the same!